lib/python/picongpu/input/parameters.py

```diff
@@ -32,8 +32,7 @@ class Parameter(object):
     this class. Either as MultiRangeSlider or as MultiSelection or as Checkbox
     depending on the 'values' (discrete) or 'range' (continuous)
 
-    Stepsizes will be handled by the widget representation, not here.q
-
+    Stepsizes will be handled by the widget representation, not here.
```

Comment:
ups, step sizes is two words

Reply:
Probably "stepping" is easier to understand?

Reply:
Agreed.